package.json

```diff
@@ -45,7 +45,7 @@
     "build": "node utils/gen.js && turbo run build",
     "vercel-build": "pnpm build && pnpm run pack && cd api && node -r ts-eager/register ./_lib/script/build.ts",
     "pre-commit": "lint-staged",
-    "test": "jest --rootDir=\"test\" --testPathPattern=\"\\.test.js\"",
+    "test": "jest --rootDir=\"test\" --testPathPattern=\"\\.test.ts\"",
```

Comment:
I don't think this is correct

Reply:
Agree w/ @styfle. There are no `.ts` files in the root `test` directory.